addon/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+// please write your own index.js that exports legit reducers
+//
+// import { combineReducers } from 'redux';
+// const foobar = (state, action) => {
+//   //reducer code here
+// };
+//
+// export default combineReducers({
+//   foobar
+// })
+import Ember from 'ember';
+
+export default function(combine) {
```

Comment:
Ah, I remember finding this in some redux source at one point. Because it's not actually intended for use we could default to the more simplistic bersuon you recommended.

When I'm back from vacation early next week I'll update this.